rm/rm-policy/rm-policy-scheduler/src/main/java/org/ow2/proactive/scheduler/resourcemanager/nodesource/policy/SchedulerAwarePolicy.java

```diff
@@ -52,38 +54,63 @@
 
     protected static Logger logger = Logger.getLogger(SchedulerAwarePolicy.class);
 
-    @Configurable
+    @Configurable(description = "url used to contact the scheduler, e.g. pnp://, pamr://")
     protected String schedulerUrl = "";
 
-    @Configurable(credential = true)
+    @Configurable(credential = true, description = "credentials used when contacting the scheduler")
     protected File schedulerCredentialsPath;
 
+    @Configurable(description = "timeout in ms for the resource manger to recover broken node source")
```

Comment:
Maybe make clearer that the next two parameters are relevant in case of the scheduler aware policy only (in description/ variable name) ?